### ISSUE TITLE
CMAP facilitation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7168]
**Preferred term label:**
CMAP facilitation

**Synonyms**
Compound muscle action potential amplitude facilitation

**Definition (free text, please give PubMed ID)**
Compound muscle action potential amplitude (CMAP) facilitation denotes an increase in tendon reflexes, strength, or CMAP amplitude after 10 seconds of maximal voluntary contraction.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:616040 

https://pubmed.ncbi.nlm.nih.gov/25192047/  6/8

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
